src/block/horizontal-scroller/style.scss

```diff
@@ -0,0 +1,21 @@
+.stk-block-horizontal-scroller:not(.editor) > .stk-block-content {
```

Comment:
Instead of targeting this for the frontend `:not(.editor)`, create an `editor.scss` (this only loads in the editor) then override the styles. We want to keep all `style.scss` lean and frontend only.